src/components/DocSearch.res

```diff
@@ -4,19 +4,40 @@ type options = {
   inputSelector: string,
 }
 
+@bs.val @bs.scope("document")
+external activeElement: option<Dom.element> = "activeElement"
+
 @bs.val @bs.scope("window")
 external docsearch: option<options => unit> = "docsearch"
 
+type keyboardEventLike = {key: string, ctrlKey: bool, metaKey: bool}
+
+@bs.val @bs.scope("window")
+external addKeyboardEventListener: (string, keyboardEventLike => unit) => unit = "addEventListener"
```

Comment:
Nitpick, not sure if you are aware of this. You can set a fixed param like `keydown` via the `as("keydown") _` placeholder:

```suggestion
external addKeydownEventListener: (@as("keydown") _, keyboardEventLike => unit) => unit = "addEventListener"
```

Reply:
will merge anyways, since this will be easy to refactor later